lms/services/grading_info.py

```diff
@@ -12,7 +12,7 @@ class GradingInfoService:
     class _ParamsSchema(PyramidRequestSchema):
         """Schema for the relevant parameters from the request."""
 
-        locations = ["form"]
+        location = "form"
```

Comment:
This change was made to lots of schemas: `webargs` now use a `location` string instead of a `locations` list of strings. In most cases we were using a single string in a list anyway so the change is simple. `BearerTokenSchema` is the one more complicated example